components/Comment/CommentEditor.tsx

```diff
@@ -68,34 +68,50 @@ const CommentEditor = ({
   }, []);
 
   useEffect(() => {
-    if (_isFocused) {
-      editorRef.current.focus();
+    if (quill) {
+      quill.on('text-change', (delta, oldDelta, source) => {
+        const nextContent = quill.getContents();
+        _setContent(nextContent);
+        contentRef.current = nextContent;
+
+        if (isQuillEmpty(nextContent)) {
+          setIsSubmitDisabled(true);
+        }
+        else {
+          setIsSubmitDisabled(false);
+        }
+      });
     }
-  }, [_isFocused]);
+  }, [quill]);
+
+  if (Quill && !quill) {
+    const MagicUrl = require('quill-magic-url').default;
```

Comment:
This section feels a bit weird to me. 

I think we currently do something like this -- possibly to get around server side rendering, but is there another way to do it?

Dynamic imports?

Reply:
Yeah I need to experiment a bit

Reply:
Ended up moving this logic to `useQuill`.
We actually want to avoid dynamic imports here since it causes a race condition. The creator of this hook recommends avoiding them.

`CommentEditor` should be cleaner. 
Verified Quill does not get remounted intermittently - only once. It now fees more lightweight.